src/container-mapping.ts

```diff
@@ -215,4 +224,70 @@ export class ContainerMapping implements IMicrosoftSecurityDevOps {
             req.end();
         });
     }
+
+    /**
+     * Queries Defender for DevOps to determine if the caller is onboarded for container mapping.
+     * @param retryCount the number of time to retry
+     * @param bearerToken the GitHub-generated OIDC token
+     * @returns a boolean Promise to indicate if the report was sent successfully or not
+     */
+    private async checkCallerIsCustomer(bearerToken: string, retryCount: number = 0): Promise<boolean> {
+        return await this._checkCallerIsCustomer(bearerToken)
+        .then(async (statusCode) => {
+            if (statusCode == 200) { // Status 'OK' means the caller is an onboarded customer.
+                return true;
+            } else if (statusCode == 403) {// Status 'Forbidden' means caller is not a customer.
+                return false;
+            } else {
+                core.debug(`Unexpected status code: ${statusCode}`);
+                if (retryCount == 0) {

```

Comment:
NIT: Might be cleaner to make this if-else retry into a separate function since its being repeated here and in line 253-259.